src/favorites/FavouriteDepartureToggle.tsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 import {StyleSheet} from '@atb/theme';
 import SvgFavorite from '@atb/assets/svg/mono-icons/places/Favorite';
 import {StoredType} from '@atb/favorites/storage';
```

Comment:
May change this import to just use `./storage`. I know this is inside same module, but if we later introduce eslint rule to assert correct imports this will probably fail.